pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -27,6 +27,7 @@
   # Feature flags
 , withAlsa ? withHeadlessDeps && stdenv.isLinux # Alsa in/output supporT
 , withAom ? withFullDeps # AV1 reference encoder
+, withAribcaption ? withFullDeps && lib.versionAtLeast version "6.1" # Portable ARIB STD-B24 Caption Decoder/Renderer
```

```suggestion
, withAribcaption ? withFullDeps && lib.versionAtLeast version "6.1" # ARIB STD-B24 Caption Decoder/Renderer
```

"portable" is irrelevant here

---

Done.